Kconfig

```diff
@@ -114,6 +114,16 @@ config DMA_FIFO_PARTITION
 
 source "src/Kconfig"
 
+menu "Pipeline"
+
+config EXTERN_XRUN_RECOV
```

Comment:
Why is it truncated, cannot we use full recovery suffix?

Reply:
I figured it was a bit long

Reply:
But now it's a bit obscure ;p

Reply:
@slawblauciak thanks